assets/javascripts/keyevent.js

```diff
@@ -1,5 +1,31 @@
-// http://stackoverflow.com/questions/5681146/chrome-10-keyevent-or-something-similar-to-firefoxs-keyevent
+//Keyboard Accessibility
+// Wrap the code in a DOMContentLoaded event listener to ensure it runs after the DOM is fully loaded
```

Comment:
I don't think we need such a comment. At least in my opinion the code speaks for itself. However, does this actually wrap the code into the listener? To me it looks like you're just passing a function with an empty body to it and the rest of the code is executed immediately. I must ask: Have you actually tested this? It would be really appreciated to extend our UI tests.

Reply:
I think anyone beginner who comes to read the code may get some help from the comments, but ok I will delete the comments.

Reply:
I would assume the following code should go into that function, but that's not the case?